docker-compose.yml

```diff
@@ -98,7 +98,7 @@ services:
       context: .
       dockerfile: ./mock-apigw/Dockerfile
     ports:
-      - 9000:8080
+      - 9010:9001
```

Comment:
Mentioned on Slack, but not sure if this can be reverted?

Reply:
This was done in error. I have reverted it back.